tutorials/plugins/android/android_plugin.rst

```diff
@@ -1,16 +1,16 @@
 .. _doc_android_plugin:
 
-Creating Android plugins
-========================
+Creating Android plugins (Godot 4.0+)
+=========================================
```

Comment:
The underline should be the same length as the title (in mono font). Same for other titles below.